padle/padle.py

```diff
@@ -97,7 +97,9 @@ async def get_menu_default_data(self, ims):
         data = {
             'dbcog': await self.get_dbcog(),
             'user_config': await BotConfig.get_user(self.config, ims['original_author_id']),
-            'today_guesses': await self.get_today_guesses(user, ims.get('current_day'))
+            'today_guesses': await self.get_today_guesses(user, ims.get('current_day')),
```

Comment:
okay so!

now that we're doing, like, A Lot with the cog in the various menus, I would suggest passing `self` as 'padle_cog' here. you'll have to go back and edit your other menu to be like padle_cog.get_today_guesses(etc) in the deserialize, but this will make it a lot easier/cleaner